changelog.d/20240221_171528_chris_endpoint_subscription_id.rst

```diff
@@ -0,0 +1,7 @@
+New Functionality
+^^^^^^^^^^^^^^^^^
+
+- Added support for the new Globus subscription management service. An endpoint can be
+  associated with a subscription group via the ``--subscription-id`` flag to
+  ``globus_compute_endpoint configure``, or via the ``subscription_id`` option in
+  ``config.yaml``.
```

Comment:
- That's `globus-compute-endpoint`, yeah?
- Please show the `subscription_id: ...` in context here.